pkg/occlient/occlient.go

```diff
@@ -681,6 +721,115 @@ func secretKeyName(componentName, baseKeyName string) string {
 	return fmt.Sprintf("COMPONENT_%v_%v", strings.Replace(strings.ToUpper(componentName), "-", "_", -1), strings.ToUpper(baseKeyName))
 }
 
+// getS2ILabelValue returns the requested S2I label value from the passed set of labels attached to builder image
+// and the hard coded possible list(the labels are not uniform across different builder images) of expected labels
+func getS2ILabelValue(labels map[string]string, expectedLabelsSet []string) string {
+	for _, label := range expectedLabelsSet {
+		if retVal, ok := labels[label]; ok {
+			return retVal
+		}
+	}
+	return ""
+}
+
+type S2IPaths struct {
```

Comment:
It is good practice to have all the definitions at the beginning of the file. It makes it easier to find them.

Reply:
Ack
Done now